modules/wkt/src/hex-wkb-loader.ts

```diff
@@ -0,0 +1,61 @@
+// loaders.gl, MIT license
+
+import type {LoaderWithParser} from '@loaders.gl/loader-utils';
+import {BinaryGeometry} from '@loaders.gl/schema';
+
+import type {WKBLoaderOptions} from './wkb-loader';
+import {WKBLoader} from './wkb-loader';
+import {VERSION} from './lib/utils/version';
+import {decodeHex} from './lib/utils/hex-transcoder';
+
+export type HexWKBLoaderOptions = WKBLoaderOptions;
+
+/**
+ * Worker loader for Hex-encoded WKB (Well-Known Binary)
+ */
+export const HexWKBLoader: LoaderWithParser<BinaryGeometry, never, HexWKBLoaderOptions> = {
+  name: 'Hexadecimal WKB',
+  id: 'wkb',
+  module: 'wkt',
+  version: VERSION,
+  worker: true,
+  category: 'geometry',
+  extensions: ['wkb'],
+  mimeTypes: [],
+  options: WKBLoader.options,
+  text: true,
+  testText: isHexWKB,
+  // TODO - encoding here seems wasteful - extend hex transcoder?
+  parse: async (arrayBuffer: ArrayBuffer) => parseHexWKB(new TextDecoder().decode(arrayBuffer)),
```

Comment:
Do you mean using `parseHexWKB(encodeHex(new Uint8Array(arrayBuffer)))`? Maybe it's not a good idea since the lookup table `0123456789abcdef` in  encodeHex() assumes lower-cases string.

Reply:
- No, because this function does no expect binary WKB, it just expects the binary representation of the string (TextEncoded if you will). 
- Most loaders.gl parsers work on binary data and the support for direct parsing from "TextDecoded" binary memory is something of an add-on.